### PR TITLE
Feature/new feature

### PR DESCRIPTION
**Pull request by Cxxshyy**
## Summary
fixed issue where options weren't seen by the user due to vs code theme.
...

### CODE REVIEW

**Comment by Cxxshyy** [comment]
This is the new version of the css styling if you can test for yourself and let me know

File: webview/features/Sidebar/Collection/SidebarCollection.js [hunk 1]
```diff
@@ -142,7 +142,8 @@ const UtilitySectionWrapper = styled.div`
     margin-top: 1rem;
     padding: 0.4rem 0.6rem;
     border-radius: 0.35rem;
-    color: white;
+    color: var(--default-text);
+    opacity: 0.78;
   }
 `;
```
Comment (unwls) [inline webview/features/Sidebar/Collection/SidebarCollection.js:149]: All changes on this PR look nice, but there still seems to be an issue with the white VSCode theme. 
Although it's not as popular as darker themes you wanna try solving this problem(Could be annoying to fix this)? 

#### Current Issue
VSCode white theme

If you are satisfied with the current change and don't wanna bother dealing with the white VSCode theme, it's totally fine!

Comment (Cxxshyy) [reply webview/features/Sidebar/Collection/SidebarCollection.js:149]: ah yes i never tested it out with white i will take a look later today.

Comment (unwls) [reply webview/features/Sidebar/Collection/SidebarCollection.js:149]: Awesome! Thanks!!

Comment (Cxxshyy) [reply webview/features/Sidebar/Collection/SidebarCollection.js:149]: Hi i never managed to find a way to fix it unfortunately so i say merge it and in the future if someone raises the issue we can take a better look

Comment (unwls) [reply webview/features/Sidebar/Collection/SidebarCollection.js:149]: Sounds good!

**Pull request merged**